docs/modules/ROOT/pages/vshn-managed/redis/plans.adoc

```diff
@@ -96,7 +95,7 @@ spec:
     name: postgres-creds
 ----
 
-IMPORTANT: Changing disk size after initial provisioning is currently not supported.
+NOTE: When changing the disk size from a plan, it will restart the redis pods. Any subsequent disk resizes should not trigger restarts.
 
```

Comment:
Does it mean that any other disk size changes after the first one will not trigger restart?

Reply:
Yes.

If you have suggestions to re-phrase it better, please feel free to suggest :D

Reply:
thinking..
does should mean that it could? :D

Reply:
I haven't observed it, but I'm also not 100% sure...

In my tests it never triggered a restart after the first time.